client/landing/stepper/declarative-flow/site-migration-flow.ts

```diff
@@ -611,6 +611,15 @@ const siteMigration: Flow = {
 						return exitFlow( addQueryArgs( { ref: 'site-migration' }, `/import/${ siteSlug }` ) );
 					}
 
+					if ( entryPoint === 'wp-admin' ) {
```

Comment:
This change is not described in the PR. What is it achieving ?

Reply:
The PR needed a rebase. This change is gone now.